CHANGELOG.md

```diff
@@ -14,6 +27,9 @@ required by the resource manager were removed from libsapi and moved into
 the 'common directory.
 - Update Linux / Unix OS detection to use non-obsolete macros.
 ### Fixed
+- Updated invalid number of handles in PolicyNvWritten and TPM2_CC_TestParms
+- Updated PlatformCommand not to send CANCEL_OFF before every command
+- Expanded TPM2B macros and removed TPM2B_TYPE1 and TPM2B_TYPE2 macros
 - Wrong return type for Tss2_Sys_Finalize (API break).
 
```

Comment:
I'd also add:
- Add const qualifiers to input pointers
- Cleaned up headers and remove a few, such as implementation.h or tpm2.h (API break)
- Change TCTI macros to CamelCase (API break)
- Rework TPMA_types to be unsigned integer with defines instead of bitfield structs (API/ABI break)
- Change Get/SetCmd/RspAuths to new parameter types (API/ABI break)

Reply:
done